Lib/multiprocessing/queues.py

```diff
@@ -20,6 +20,8 @@
 
 from queue import Empty, Full
 
+import _multiprocessing
```

Comment:
This could still use a comment.

Reply:
I reverted the removal of this import because you're right: it's a C extension; I'm not a multiprocessing expert; and there's a chance that there's some subtle side effect here that I'm not aware of from importing `_multiprocessing` here. But that doesn't mean that I know _why_ it's there, so I'm not sure what I'd put in a comment necessarily.

Digging through `git blame`, it seems the import was first introduced in https://github.com/python/cpython/commit/e711cafab13efc9c1fe6c5cd75826401445eb585, in 2008. At that point in time, the import _was_ explicitly used, on this line here: https://github.com/python/cpython/commit/e711cafab13efc9c1fe6c5cd75826401445eb585#diff-ac78ddc352ec78ebe64a4ab3b7ffd9e41f91291b3b856fb2e30849c118f68935R34

The usage of that import in `multiprocessing.queues` was removed in https://github.com/python/cpython/commit/7ecfc82edbe5ccb125f53b92447abd4bc155ba1c, in 2015, but the import statement was not removed.

This history leads me to conclude that this likely is, in fact, a "true positive", and that the import likely is unused here, and can be removed. But I'm not a multiprocessing expert. So I can't say for sure. And I'm not sure what I'd put in a comment.

Reply:
Honestly, I'm not a multiprocessing expert either. So I withdraw the request for a comment. But I also don't think either of us knows whether that import is truly redundant. So let's just keep it unchanged, and let someone else figure it out in the future.

Reply:
> So let's just keep it unchanged, and let someone else figure it out in the future.

SGTM